pkg/config/config.go

```diff
@@ -376,6 +376,8 @@ type EngineConfig struct {
 	// RemoteIdentity key file for RemoteURI
 	RemoteIdentity string `toml:"remote_identity,omitempty"`
 
+	MachineMode bool `toml:"machine_mode,omitempty"`
```

Comment:
We need an engine option for passing from podman In my podman PR I utilize this option but it is not available as a flag to users

Reply:
not sure, I think it may be necessary? I will look around. Either way, I think we should deal with things like this later (as long as it cannot be set via a flag)

Reply:
Adding fields to file formats is kinda forever, so it shouldn’t happen if we are not sure.

Reply:
I think I can remove it as long as this is ok to always be false
https://github.com/containers/common/blob/55b1e5177f486bfc3b6e9286815bd69a36016200/pkg/config/config.go#L1264-L1266

which I think is reasonable.

Reply:
I think so as well, unless machine creation sets these fields.